paymentsheet/res/values/themes.xml

```diff
@@ -37,10 +37,4 @@
         <item name="colorPrimary">@color/stripe_paymentsheet_form</item>
         <item name="colorAccent">@color/stripe_paymentsheet_form</item>
     </style>
-
-    <style name="StripeLinkBaseTheme" parent="@android:style/Theme.Translucent">
-        <item name="android:windowContentOverlay">@null</item>
-        <item name="android:windowNoTitle">true</item>
-        <item name="windowNoTitle">true</item>
-    </style>
```

Comment:
We're basically using the `StripePaymentSheetDefaultTheme` now. Let me know if you prefer a copy that's under `StripeLinkDefaultTheme` instead.

Reply:
Let's use this for now!